google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/mapping/basic/ParentTest.java

```diff
@@ -97,11 +98,10 @@ public void setCity(City city) {
 
 	//tag::mapping-Parent-example[]
 
-	@Entity(name = "City")
+	@Entity(name = "City_BasicParentTest")
 	public static class City {
 
 		@Id
-		@GeneratedValue
```

Comment:
does the generated value cause problems? Do some tests rely on a static id (the sequence table being reset each test) ?

For those cases I changed the queries instead of removing the generated value, but either way works .

Reply:
Right, there were no problems using GeneratedValue; however, the test was doing a `entitymanager.getById` with `id == 1` and this assumes the sequence table being reset with each test. So I just removed generatedValue so that we can set the ID manually; figure this would be easier.

Reply:
Yea I came across a few of those situations, but I replaced `1` with `(select min(id) from hibernate_sequence)`. Anyways LGTM